src/test/java/uk/gov/hmcts/juror/api/moj/controller/response/CourtLocationsListDtoTest.java

```diff
@@ -15,19 +15,20 @@
 @RunWith(SpringRunner.class)
 public class CourtLocationsListDtoTest {
 
-    private CourtLocation createCourtLocation(String locationCode, String name, String attendacneTime) {
+    private CourtLocation createCourtLocation(String locationCode, String name, String attendanceTime, String owner) {
```

Comment:
Would be better to directly pass LocalTime instead of String here

Reply:
updated